OpenSim/Common/Test/testTRCFileAdapter.cpp

```diff
@@ -145,7 +145,7 @@ int main() {
         std::cout << "  " << filename << std::endl;
         TRCFileAdapter trcfileadapter{};
         try {
-            auto table = trcfileadapter.read(filename);
+            auto table = trcfileadapter.readFile(filename);
```

Comment:
`read` should work here since this is an instance method.

Reply:
Not quite, since read returns a map of tables while the convenience method returns a single table that can be directly written to a file.